examples/fscloud/variables.tf

```diff
@@ -63,3 +63,9 @@ variable "kms_key_crn" {
   type        = string
   description = "The root key CRN of a Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. See https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-hpcs&interface=ui for more information on integrating HPCS with Event Streams instance."
 }
+
+variable "metrics" {
```

Comment:
same as https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/pull/319/files#r1828344086

Reply:
Same problem here, examples/fscloud is using standard plan.

However modules/fscloud is using the enterprise plan, so it could include the options. Should it, though? All the other variables in modules/fscloud (topics, resource tags, schemas, etc.) default to no value.

Reply:
created tracking issue https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/issues/331